include/tdi/common/tdi_utils.hpp

```diff
@@ -285,9 +285,11 @@ static inline tdi_status_t keyFieldSafeGet(
       );
       return TDI_INVALID_ARG;
     }
-    if ((*key_field)->isPtrGet() && field_value->size_ < 64) {
+    /* If field size less than or equal to 8 and is_ptr is true,
+     * should not use byte arrays */
+    if ((*key_field)->isPtrGet() && field_value->size_ <= 8) {
       LOG_ERROR(
-          "%s:%d Field size is greater than 64 bits. Please use byte arrays in "
+          "%s:%d Field size is greater than 64 bits. Please don't use byte arrays in "
```

Comment:
The comment and the print need to be reverted back. If is_ptr is true that means that the actual field size is greater than 8 bytes.  The value of field_value->size happens to be <=8 when the user uses the setValue/getValue uint64_t version. And if that happens then the user is supposed to be using the bytes version

Reply:
Yes. done
thanks